easeljs/easeljs.d.ts

```diff
@@ -161,8 +161,8 @@ declare module createjs {
         tickChildren: boolean;
 
         // methods
-        addChild(...child: DisplayObject[]): DisplayObject;
-        addChildAt(child: DisplayObject, index: number): DisplayObject; // add this for the common case
+        addChild<T extends DisplayObject>(child: T, ...restChildren: DisplayObject[]): T;

```

Comment:
this line seems incorrect. 
The return value should be the last element.

> Returns:
> DisplayObject: The child that was added, or the last child if multiple children were added.

http://www.createjs.com/docs/easeljs/classes/Container.html#method_addChild